docs/building-connectors.md

```diff
@@ -1,114 +1,136 @@
-# Building your own Reshuffle Connector 
-Reshuffle comes with several connectors that connect to common systems. 
+# Building Reshuffle Connectors
+The Reshuffle connectors ecosystem comprises several connectors to common systems. 
 It is also extendable and lets you create your own connector.
- You will want to create a connector when you need to integrate with a homegrown system,
-  or a service that does not have a connector yet.
+
+Building your own connector may be required in some cases. For example - when you need to integrate with a homegrown system, 
+or in cases when you may be using a web based service that does not have a connector yet.
+
 
 The easiest way to create a new connector is you use the template in our
 reshuffle-example-connector repo (coming soon!). You can also install and
-extend the reshuffle-base-connector repo without using our example. Lastly,
-if you like to do things all by yourself, you just need to create a class
+extend the reshuffle-base-connector repo without using our example. 
+
+Lastly, if you like to develop things all by yourself from scratch, you just need to create a class
 and implement the following methods.
 ### The Connector class
 ```js
-class MyConnector{
+class MyConnector {
     
-constructor(connectorOptions, id) {...}
+  constructor(options, id) {...}
+
+  onStart(app) {...}
 
-start(app) {...}
+  onStop() {...}
```

Comment:
I think need to have both the stop and start and the onstop and on start. You need to implement the former if you are not inheriting from base connector.  Especially in light of the paragraph above.

Reply:
Good point. If you don't use `BaseConnector` then you need start and stop and if you do use it you need `onStart` and `onStop`. What would be a clear way to explain this?